phases/repo.go

```diff
@@ -1,9 +1,104 @@
 package phases
 
-import "fmt"
+import (
+	"fmt"
+	"strings"
 
-// Repo ...
-func Repo() (string, error) {
-	fmt.Println("SetRepo")
-	return "", nil
+	"github.com/bitrise-io/go-utils/command"
+	"github.com/bitrise-io/go-utils/errorutil"
+	"github.com/bitrise-io/go-utils/log"
+)
+
+type urlParts struct{
+	host string
+	owner string
+	slug string
+}
+
+func parseURL(cloneURL string) urlParts {
+	parts := strings.SplitAfter(cloneURL, "https://")
+	if len(parts) > 1 {
+		// e.g. cloneURL=https://github.com/bitrise-io/go-utils.git
+		parts = strings.Split(parts[1], "/")
```

Comment:
NOTE: For this you can use the URL parse method:
https://golang.org/pkg/net/url/#Parse

Reply:
Unfortunately, I can't use it for extracting all the data, so that's why I didn't want to mix it if I'd have to do parse it manually anyway.